main.go

```diff
@@ -65,6 +67,7 @@ func main() {
 		log.Err(err).Msg("can't resolve environment variables")
 		os.Exit(1)
 	}
+	log.Debug().Msgf("resolved config:\n%s", str.ToString(config))
```

Comment:
Consider removing newline (`\n`) from log message:
```suggestion
	log.Debug().Msgf("resolved config: %s", str.ToString(config))
```